src/GraphQL/Utilities/ServiceProviderExtensions.cs

```diff
@@ -28,7 +28,7 @@ public static object GetRequiredService(this IServiceProvider provider, Type ser
             if (service != null)
                 return service;
 
-            throw new InvalidOperationException($"Required service for type {serviceType} not found");
+            throw new InvalidOperationException($"No service for type '{serviceType}' has been registered.");
```

Comment:
Copied from MS implementation.

Reply:
I suggest we use the friendly name extension method we have for better support of generic types

Reply:
```suggestion
            throw new InvalidOperationException($"No service for type '{serviceType.GetFriendlyName()}' has been registered.");
```

Reply:
ok